### PR TITLE
ID3v1 track number and comment

### PR DESCRIPTION
**Pull request by upperstream**
This patch corrects the behaviour of ID3v1 (and v1.1) track number and comment retrieval.
Simple tests and test data come along with.
Thanks.

### CODE REVIEW

File: id3v1_test.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	for _, name := range []string{
+		"id3v1_test/sample_usascii_v1.1.mp3",
+		"id3v1_test/sample_ms932_v1.1.mp3",
+		"id3v1_test/sample_utf8_v1.1.mp3"} {
+		doTest(name, 1, 28, t)
+	}
+}
+
+func doTest(name string, track int, length int, t *testing.T) {
+	mp3 := MustAsset(name)
```
Comment (wader) [inline id3v1_test.go]: seems missing hmm

Comment (upperstream) [inline id3v1_test.go]: An intermediate Go source file that contains MP3 files for test was missing, which should be generated in advance of testing.  I will re-push the branch to add the missing file, or I can create another pull request with a clean branch containing everything necessary.

File: id3v1_test.go [hunk 2]
```diff
@@ -5,11 +5,12 @@
 package tag
 
 import (
+	"./id3v1_test"
```
Comment (dhowden) [inline id3v1_test.go]: Update this so that it's not a relative path (and put it below the std lib imports).

**Comment by dhowden** [comment]
Thanks for the pull request.  I have very few ID3v1 files so it's great to have more test cases (and fixes!).

File: id3v1_test.go (hunk 2 above)
Comment (upperstream) [inline id3v1_test.go]: Thank you for pointing it out.  I will update the branch and push it later.

**Comment by dhowden** [comment]
Thanks.  Rebased and merged as 0b71236

**Pull request closed without merging**